### ISSUE TITLE
Change IPHC code to use underscores not hooksCount etc.

### ISSUE THREAD

**Issue by andrew-edwards** [pbs-assess/gfplot#38]

**Comment by seananderson** [comment]
I think `lintr::lint_package()` will warn you about things like this to make them easy to find. If not that one then there's another package styling function I've used before.

**Comment by andrew-edwards** [comment]
My final results actually have things like E_it as column names (as underscores to match the math notation).  

In the commit comment you mentioned it's not a huge deal for internal code - so okay for calculations within functions that don't end up being output (though I would try and make it all consistent).

**Comment by seananderson** [comment]
Yes, and that seems like a time when clarity of notation to match the math (the capital E) trumps the style guide.
